docs/source/email_setup.rst

```diff
@@ -0,0 +1,115 @@
+
+``smpt_password`` The password associated with the ``smtp_username``. **NOTE**
+If you are using two-factor authentication, see :ref:`two-factor-auth`, below.
+
+``dallinger_email_address`` The email address to be used as the "from" address
+outgoing email notifications. For Gmail accounts, this address is likely to be
+overwritten by the Google SMTP server. See :ref:`from-address-rewrite` below.
+
+``contact_email_on_error`` Also an email address, and used in two ways:
+
+1. It serves as the recipient address for outgoing notifications
+2. It is displayed to experiment participants on the error page, so that
+   they can make inquiries about compensation
+
+
+Pitfalls and Solutions
+~~~~~~~~~~~~~~~~~~~~~~
+
+A few other things which may get in the way of sending email successfully, or
+cause things to behave differently than expected:
+
+
+.. _two-factor-auth:
+
+Two-Factor Authentication
+"""""""""""""""""""""""""
+
+Having two-factor authentication enabled for the outgoing email account will
+prevent dallinger from sending email without some additional steps. Details
+instructions are provided for Gmail, below. Other email services which support
+two-factor authentication may provide equivalent solutions for marking an app
+as safe to send email.
+
+Working with Google/Gmail Two-factor Authentication
+'''''''''''''''''''''''''''''''''''''''''''''''''''
+If you are using Gmail with two-factor authentication, we recommend that you set
+up an application-specific password (or, as Google calls it, an "App password")
+specifically for Dallinger. You can set one up following these instructions
+(adapted from `here <https://www.lifewire.com/get-a-password-to-access-gmail-by-pop-imap-2-1171882>`_):
+
+#. Log into your Gmail web interface as usual, using two-factor authentication if
+   necessary.
+#. Click your name or photo near your Gmail inbox's top right corner.
+#. Follow the *Google Account* link in the drop-down/overlay that appears.
+#. Click *Signing in to Google* in the *Sign-in & security* section.
+#. Under the *Password & sign-in method* section, click *App passwords*.
+   (If prompted for your Gmail password, enter it and click *Next*.)
+#. Select *Other (custom name)* in the *Select app* drop-down menu.
+   Enter *Dallinger outgoing mail* or another descriptive name so you'll recognize
+   what it's for when you view these settings in the (potentially distant) future.
+#. Click *Generate*.
+#. Find and immediately copy the password under *Your app passwords*. Type or paste the
+   password into the `.dallingerconfig` file in your home directory.
+   You will not be able to view the password again, so if you miss it, you'll
+   need to delete the one you just created and create a new one.
+#. Click *Done*.
+
+Firewall/antivirus
+""""""""""""""""""
+When developing locally, antivirus or firewall software may prevent outgoing
+email from being sent, and cause dallinger to raise a `socket.timeout` error.
```

Comment:
Maybe keep Dallinger as the name of the app consistently as "Dallinger" vs "dallinger" throughout the document?
Occurs in several places.